node-graph/gcore/src/vector/generator_nodes.rs

```diff
@@ -6,19 +6,96 @@ use bezier_rs::Subpath;
 
 use glam::DVec2;
 
-pub struct UnitCircleGenerator;
+#[derive(Debug, Clone, Copy)]
+pub struct UnitCircleGenerator<Radius> {
```

Comment:
Where you have mentioned unit in these node names is outdated (as the output can be of any size).